XCUITests/SearchTest.swift

```diff
@@ -218,4 +218,38 @@ class SearchTests: BaseTestCase {
         waitForExistence(app.textFields["url"], timeout: 20)
         waitForValueContains(app.textFields["url"], value: "google")
     }
+    
+    func testSearchIconOnAboutHome() {
+        waitForTabsButton()
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitForTabsButton()
+        
+        // Search icon is displayed.
+        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
+        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Search")
+        XCTAssertTrue(app.buttons["TabToolbar.stopReloadButton"].exists)
+        app.buttons["TabToolbar.stopReloadButton"].tap()
+
+        let addressBar = app.textFields["address"]
+        XCTAssertTrue(addressBar.value(forKey: "hasKeyboardFocus") as? Bool ?? false)
+        XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
+    
+        app.textFields["address"].typeText("www.google.com\n")
+        waitUntilPageLoad()
+
+        // Reload icon is displayed.
+        waitForExistence(app.buttons["TabToolbar.stopReloadButton"])
+        XCTAssertEqual(app.buttons["TabToolbar.stopReloadButton"].label, "Reload")
+        app.buttons["TabToolbar.stopReloadButton"].tap()
```

Comment:
There is an action defined to reload a web page: `navigator.performAction(Action.ReloadURL)`

You may have to for the graph to know where you are by adding before the action this line, where the action is called from: navigator.nowAt(BrowserTab)